sous-chef/pyproject.toml

```diff
@@ -35,5 +37,6 @@ build-backend = "poetry.core.masonry.api"
 [tool.pytest.ini_options]
 markers = [
     "gmail: marks gmail tests which requires frequent user validation for oauth",
+    "gsheets: marks tests using gsheets",
```

Comment:
@AkagiShigeru Are there any other tests that you can think of that have an external dependency? Are there any tests that I missed labeling?

I'd propose that we continue to label any such tests with such a dependency. That way is something goes down, we can maybe more easily see what happened.

The OpenFoodFacts API doesn't require a token, but would it be worthwhile to label them? (e.g. in case, their service is down for a while)

Reply:
I think you marked all the relevant tests. It looks like a good practice to me, I wasn't aware of this functionality.
I think marking the OpenFoodFacts ones makes sense as it's still using an API (even without an access token).